prime-router/src/test/kotlin/DocumentationTests.kt

```diff
@@ -222,13 +222,53 @@ $documentation
 
 **Value Sets**
 
-Code | Display
----- | -------
-&#62;|Above absolute high-off instrument scale
+Code | Display | System
+---- | ------- | ------
+&#62;|Above absolute high-off instrument scale|HL7
 
 ---
 """
         val actual = DocumentationFactory.getElementDocumentation(elemWithTypeCode)
         assertThat(actual).isEqualTo(expected)
     }
+
+    @Test
+    fun `test documentation for element with valueSet table generation`() {
+
+        // Test case that contains element.system = HL7 and element.ValueSet.Values.system = LOINC
+        val valueSetA = ValueSet(
+            "a",
+            ValueSet.SetSystem.HL7,
+            values = listOf(
+                ValueSet.Value(
+                    ">", "Above absolute high-off instrument scale",
+                    system = ValueSet.SetSystem.LOINC
+                )
+            )
+        )
+        val elemWithValuesSetValues = Element(name = "a", type = Element.Type.CODE, valueSetRef = valueSetA)
+        var expected = """
```

Comment:
We shouldn't be using `var` unless it's needed. please change it to val.

Reply:
Done.